e2e/pages/noticeOfChange.page.js

```diff
@@ -0,0 +1,35 @@
+const { I } = inject();
+
+module.exports = {
+  noticeOfChange: 'Notice of change',
+  fields: {
+    caseRefSearch: '#caseRef',
+    applicantName: '#applicantName',
+    respondentFirstName: '#respondentFirstName',
+    respondentLastName: '#respondentLastName',
+    confirmNoC: '#affirmation',
+    notifyEveryParty: '#notifyEveryParty',
+  },
+
+  navigate(){
+    I.click(this.noticeOfChange);
+  },
+
+  enterCaseReference(caseReference) {
+    I.fillField(this.fields.caseRefSearch, caseReference);
+  },
```

Comment:
As this page doesn't have an accessibility check  it would need an `await I.runAccessibilityTest();`, e.g.
```suggestion
  async  enterCaseReference(caseReference) {
    await I.runAccessibilityTest();
    I.fillField(this.fields.caseRefSearch, caseReference);
  },
```